lib/exercises.js

```diff
@@ -1,17 +1,68 @@
 // This method will return an array of arrays.
 // Each subarray will have strings which are anagrams of each other
-// Time Complexity: ?
-// Space Complexity: ?
+// Time Complexity: O(m) m is all of the characters in all of the words
```

Comment:
Usually when doing time complexity like this, variables can't really be assigned for the entire set of input, it only really makes sense to talk about the effect each element of the input has.

So it would be more common to say that `m` is the number of characters per word, and then using `n` as the number of words in the list, we'd get O(n*m).

However, knowing that the words are English words, we can make a simplifying assumption. English words tend not to get so long, about 5 letters on average. With that knowledge, we can tell that the number of words in the list is going to have much more of an effect than the number of letters in the word, and just say it's O(n).

Reply:
lol I did have this has the time complexity and then talked myself out of it, but thank you for reaffirming my original assumptions.🙃